src/main/java/baseball/dto/Score.java

```diff
@@ -0,0 +1,74 @@
+package baseball.dto;
+
+import static baseball.enums.Message.BALL;
+import static baseball.enums.Message.STRIKE;
+
+public class Score {
+    private int ballCount;
+    private int strikeCount;
+
+    private Score(ScoreBuilder builder) {
+        this.ballCount = builder.ballCount;
+        this.strikeCount = builder.strikeCount;
+    }
+
+    public static ScoreBuilder builder() {
+        return new ScoreBuilder();
+    }
+
+    public int getBallCount() {
+        return ballCount;
+    }
+
+    public int getStrikeCount() {
+        return strikeCount;
+    }
+
+    public void increaseBallCount() {
+        this.ballCount++;
+    }
+
+    public void increaseStrikeCount() {
+        this.strikeCount++;
+    }
+
+    public String getScoreBoardMessage() {
+        return (getBallCountMessage() + " " + getStrikeCountMessage());
+    }
+
+    public String getBallCountMessage() {
+        return (this.getBallCount() != 0) ? this.getBallCount() + BALL.getMessage() : "";
+    }
+
+    public String getStrikeCountMessage() {
+        return (this.getStrikeCount() != 0) ? this.getStrikeCount() + STRIKE.getMessage() : "";
+    }
```

Comment:
다음에는 삼항 연산자는 지양해보면 어떨까요!?
삼항 연산자는 가독성에 좋지 않은 것 같아요.
한 줄에 너무 많은 구문이 들어가 있어 파악하기 쉽지 않은 것 같습니다!

Reply:
@wooteco-daram 삼항연산자 자체가 가독성이 좋지 않다는 것에는 어떤 이유에서 일까요? 
삼항 연산자가 중첩되거나 내부에 문(statement)를 담고 있지 않다면 오히려 간결해서 한눈에 들어온다고 생각합니다.

한줄에 너무 많은 내용이 들어가서 불편하면 줄바꿈으로 처리해보면 어떨까요?

```suggestion
    public String getBallCountMessage() {
        return (this.getBallCount() != 0) 
            ? this.getBallCount() + BALL.getMessage()
            : "";
    }

    public String getStrikeCountMessage() {
        return (this.getStrikeCount() != 0)
            ? this.getStrikeCount() + STRIKE.getMessage()
            : "";
    }
```